pkg/kubelet/dockershim/network/hostport/hostport_manager.go

```diff
@@ -33,8 +33,12 @@ import (
 	utiliptables "k8s.io/kubernetes/pkg/util/iptables"
 	utilnet "k8s.io/kubernetes/pkg/util/net"
 	"k8s.io/utils/exec"
+	"os"
 )
 
+// conntrack will be downloaded to this path by the configure.sh script on the node
+const customConntrackPath = "/home/kubernetes/bin/conntrack"
```

Comment:
Is it possible to read it in from an environment variable?

For instance, assume it is under `$KUBE_BIN/conntrack`. Instead of hard coded into OSS kubelet.

Reply:
Good idea. Those environment variables were not set inside kubelet either. I found that it was possible to set env variables for kubelet in the /etc/default/kubelet file, so I ended up setting $PATH.